src/main/scala/devices/debug/DMI.scala

```diff
@@ -79,7 +79,18 @@ class ClockedDMIIO(implicit val p: Parameters) extends ParameterizedBundle()(p){
 
 class DMIToTL(implicit p: Parameters) extends LazyModule {
 
-  val node = TLClientNode(Seq(TLClientPortParameters(Seq(TLClientParameters("debug")))))
+  val node = TLClientNode(Seq(TLClientPortParameters(Seq(TLClientParameters("debug",
+  knownToEmit = Some(TLClientEmissionSizes(
+    emitsAcquireT = TransferSizes.none,
+    emitsAcquireB = TransferSizes.none,
+    emitsArithmetic = TransferSizes.none,
+    emitsLogical = TransferSizes.none,
+    emitsGet = TransferSizes(4, 4),
+    emitsPutFull = TransferSizes(4, 4),
+    emitsPutPartial = TransferSizes.none,
+    emitsHint = TransferSizes.none
```

Comment:
I think for brevity this is a fine spot to rely on the default parameter values of `TransferSizes.none` and only call out the specific types that are in fact emitted.